src/ZfcTwig/Twig/FallbackFunction.php

```diff
@@ -5,7 +5,7 @@
 use Twig_Function;
```

Comment:
you can also change `Twig_Function` to `Twig_SimpleFunction` so we can also remove the `\` before the `Twig_SimpleFunction` class